pychromecast/__init__.py

```diff
@@ -171,8 +179,23 @@ def __init__(self, host, port=None, device=None, **kwargs):
         self.status = None
         self.status_event = threading.Event()
 
+        self.browser = None
+        self.tracker = None
+
+        self.setup(host, port, self.device, self.kwargs)
+
+    def setup(self, host, port, device, kwargs):
+        """
+        Method for setting up, and resetting the chromecast object
+        """
+        tries = 2 if self.cast_type == CAST_TYPE_GROUP \
```

Comment:
Why is tries bigger for a group? We should store these values in the constructor.

Also, please rewrite as normal if statement instead of inline if statement. It is confusing to read.